<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autoclick 0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_Start">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Stop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>127</y>
      <width>89</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> (Ctrl + Shift + S)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>127</y>
      <width>87</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>(Ctrl + Shift + P)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Confirmar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbox_Keys">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LEFT MOUSE BUTTON</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RIGHT MOUSE BUTTON</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KEYBOARD_A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KEYBOARD_B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KEYBOARD_C</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="edit_Interval">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Interval:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Seconds</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>474</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
